src/main/java/com/nike/cerberus/operation/core/CreateDatabaseOperation.java

```diff
@@ -85,10 +88,18 @@ public void run(final CreateDatabaseCommand command) {
         final TypeReference<Map<String, String>> typeReference = new TypeReference<Map<String, String>>() {};
         final Map<String, String> parameters = cloudFormationObjectMapper.convertValue(databaseParameters, typeReference);
 
-        cloudFormationService.createStack(Stack.DATABASE, parameters, true, command.getTagsDelegate().getTags());
+        String stackId = cloudFormationService.createStack(Stack.DATABASE, parameters, true,
```

Comment:
Not sure I understand this change.  Didn't createStack already wait for the change to complete?

Reply:
no, it just returns a stack id, there is a polling method that you can call to wait and poll for a status

Reply:
Okay